rosdep/python.yaml

```
@@ -8519,6 +8535,18 @@ python3-weasyprint-pip:
   ubuntu:
     pip:
       packages: [weasyprint]
+python3-webargs:
+  arch: [python-webargs]
+  debian:
+    pip:
+      packages: [webargs]
+    sid: [python3-webargs]
```

```suggestion
    '*': [python3-webargs]
```

Switching this to * for bookworm and newer